### ISSUE TITLE
[API] unable to get all documents related to a person

### ISSUE THREAD

**Issue by AndreSchwarzer** [LilithWittmann/credentials_dev_public#4]
# Get all documents related to a person [/v1/people/{person_id}/documents/](https://sandbox.api.credentials.dev/docs#/v1/get_person_documents)

```shell
curl -X 'GET' \
  'https://sandbox.api.credentials.dev/v1/people/34a36028-dddc-477f-9676-ceb75089658a/documents/' \
  -H 'accept: application/json' \
  -H 'X-API-Key: REDACTED'
```

The requests results in a 500 `Internal Server Error` response. Requesting a person's details, contains a list of its documents.

```shell
curl -X 'GET' \
  'https://sandbox.api.credentials.dev/v1/people/34a36028-dddc-477f-9676-ceb75089658a/' \
  -H 'accept: application/json' \
  -H 'X-API-Key: REDACTED'
```

```json
{
  "created_at": "2022-12-27T13:09:31.443720",
  "id": "34a36028-dddc-477f-9676-ceb75089658a",
  "tenant_id": "b25eb48b-abbd-46c5-a1bc-146fab1a1f58",
  "external_identifier": "string",
  "documents": [
    {
      "created_at": "2022-12-27T13:33:50.098073",
      "id": "fe3c44b4-86c1-4cd7-9335-bd183b4f48d1",
      "external_identifier": "string",
      "validations": []
    }
  ]
}
```

**Comment by LilithWittmann** [comment]
Fixed in 61d5f19.

**Comment by AndreSchwarzer** [comment]
confirmed, works
